src/app/story/directives/status-control.component.ts

```diff
@@ -1,28 +1,37 @@
 import { Component, DoCheck, Input, Output, EventEmitter, ViewChild, ElementRef } from '@angular/core';
```

Comment:
this is just a lot of auto formatting with the ViewChild update, but it did actually catch some `,`/`;` syntax issues 😬

Reply:
Nice to have this stuff linting, though.  But you are reminding me of all my poor syntax decisions in this repo ... i just love trying to fit everything on 1 line in the `template`!